app/views/NonTaxToTaxProgressView.scala.html

```diff
@@ -0,0 +1,83 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.declaration.routes._
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.auth.core.AffinityGroup
+@import uk.gov.hmrc.auth.core.AffinityGroup.{Agent, Organisation}
+@import viewmodels.Task
+
+@this(
+    appConfig: FrontendAppConfig,
+    main_template: MainTemplate
+)
+
+@(identifier: String,
+    identifierType: IdentifierType,
+    mandatory : List[Task],
+    affinityGroup : AffinityGroup,
+    nextUrl : String,
+    isAbleToDeclare: Boolean,
+    closingTrust: Boolean)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("transitionProgress.title")
+) {
+
+   @components.back_link()
+
+    @components.heading_with_caption(headingKey = "transitionProgress.heading", caption = s"$identifierType.caption", captionParam = identifier)
+
+    <p>@messages("transitionProgress.p1")</p>
+
+    <h2>@messages("transitionProgress.subHeading")</h2>
+
+    <p>@messages("transitionProgress.p2")</p>
+
+    @components.bullet_list("transitionProgress", Seq("bullet1", "bullet2"))
+
+    @if(affinityGroup == AffinityGroup.Agent) {
+        <p>
+            <a href="@{appConfig.agentOverviewUrl}">@messages("transitionsProgress.return.link")</a>
+        </p>
+    }
+
+    @components.warning("transitionProgress")
+
+    <p>@messages("transitionProgress.p3") @components.link(appConfig.loginContinueUrl, "return-to-service", "transitionProgress.p3.link", false)</p>
+
+    <p>@messages("transitionProgress.p4")</p>
+
+    @components.task_list("mandatory", mandatory, messages("transitionProgress.subHeading.2"))
+
+    @if(isAbleToDeclare) {
+        <h2 id="summary-heading" class="app-task-list__section">@messages("transitionProgress.subHeading.4")</h2>
+        <div id="summary-paragraph" class="paragraph">
+            <p>@messages("transitionProgress.p5")</p>
+            <p>@messages("transitionProgress.p6")
+                @components.link(controllers.print.routes.PrintMaintainDraftAnswersController.onPageLoad().url, "print-and-save", "transitionProgress.printsave.link", openInNewWindow = false)
+            </p>
+        </div>
+        <h2 id="summary-heading-2">@messages("transitionProgress.subHeading.5")</h2>
+        <div id="summary-paragraph-2" class="paragraph">
+            <p>@messages("transitionProgress.p7") <a href="@{appConfig.sa900Link}" target="_blank">@messages("transitionProgress.sa900.link")</a>.</p>
+        </div>
+        <div class="section">
+            <p>@components.button_link(messages("taskList.summary.continue"), nextUrl)</p>
+        </div>
```

Comment:
Feels like there's a lot of unnecessary `div`s here just for the sake of having an id to check for in the unit tests. Could be a bit tidier to have something like https://github.com/hmrc/trusts-frontend/blob/master/app/views/register/TaskListView.scala.html and then use assertContainsText rather than id in the tests